### PR TITLE
Fix funtests for pyamqp and django

### PR DESCRIPTION
**Pull request by alanjds**
The funtests are broken, at least for Python 3.4.

Tested on Python 2.7 and 3.4 on my machine

### CODE REVIEW

File: funtests/transport.py [hunk 1]
```diff
@@ -21,13 +21,19 @@
 else:
     from sha import new as _digest  # noqa
 
+if not hasattr(string, 'letters'):
```
Comment (ask) [inline funtests/transport.py]: What version of Python does not have string.letters?  Kombu 3.0 supports 2.6 and up, and as far as I can tell it's present.

Comment (alanjds) [inline funtests/transport.py]: At least Python 3.4.3 have it not, but both Python 2.7.11 and 3.5 have it!

```bash
$ python -V
Python 3.4.3
$ python -c "import string; string.letters"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: 'module' object has no attribute 'letters'
```

**Comment by auvipy** [comment]
@alanjds could you fix the build failures?

File: funtests/transport.py (hunk 1 above)
Comment (ask) [inline funtests/transport.py]: Ah, so it was removed in Python 3!  Seems the reasoning behind the change is that `string.letters` actually varies defined on the locale set (which is news to me), so string.ascii_letters is probably what we should use always.

**Comment by ask** [comment]
I merged it manually, since I was so slow at responding to this, but only against master.  I'm not sure there is any rush merging with 3.0 and we really need to get this stuff out the door.

**Pull request closed without merging**